cmd/prometheus-config-reloader/main.go

```diff
@@ -98,6 +156,15 @@ func main() {
 		})
 	}
 
+	if *listenAddress != "" {
+		g.Add(func() error {
+			level.Info(logger).Log("msg", "Starting web server for metrics", "listen", *listenAddress)
+			http.Handle("/metrics", promhttp.HandlerFor(r, promhttp.HandlerOpts{Registry: r}))
+			return http.ListenAndServe(*listenAddress, nil)
```

Comment:
Is there a plan to monitor these metrics in the operator bundle/examples?

Reply:
These metrics are exposed by the config reloader so the service monitor and alerts should probably live elsewhere than this repository?

Reply:
Seems like a follow-up would be in kube-prometheus as we will need to adjust 2 services and 2 serviceMonitors.
It would be best to include alerts for this in prometheus-operator mixin. But this would be best to do in a follow-up too.